rules/typescript.js

```diff
@@ -48,17 +56,15 @@ const typescriptRules = {
     },
     {
       selector: 'parameter',
-      format: ['camelCase'],
+      format: ['camelCase', 'PascalCase'],
```

Comment:
Is this just any function parameter?

Reply:
yes. i had to add it for this line: `import('./webcast-settings').then(({ WebcastSettings }) => ({`